### ISSUE TITLE
Not seeing errors that are reported by ember-cli-template-lint

### ISSUE THREAD

**Issue by justinaray** [binhums/linter-ember-template#2]
I am seeing errors in the console, but my template looks fine in atom.  I have restarted, tried to toggle on/off, changed my grammar type, but nothing I do shows any template warnings/errors.

I have linter installed and get other linting errors (jscs, jshint, etc.)

Are there any other dependencies I may need or packages that may be conflicting?

Thanks!

Atom 1.9.0-beta1

**Comment by binhums** [comment]
Hi @justinaray, thanks for raising. I'll have a look today to see if I can make a fix.

My initial guess is that it's not being started automatically and might need toggling to turn on. These rules should turn on automatically.

**Comment by justinaray** [comment]
Thanks for the quick reply @binhums.

I actually have gotten a bit farther now.  I set the grammar to HTML (Handlebars) and now I'm getting a linter error.  However, it appears to be stuck with an old error and the linting is not updating.  I did get a `Uncaught TypeError: Invalid Point: (NaN, NaN)` error from the linter package.

Going to restart atom to see if that will fix the state issue.

**Comment by binhums** [comment]
If you find a fix in your explorations do feel free to submit a pull request

**Comment by binhums** [comment]
Hi @justinaray, I've published v0.1.2 which should solve the initial problem of the addon not starting.

Could you let me know if you're still experiencing the Uncaught TypeError and we can spin off a separate issue. I may need an example snippet of template code that exhibits the error from you.

**Comment by binhums** [comment]
The output of command line would also be useful for that point.

**Comment by justinaray** [comment]
Thanks for the init fix @binhums and sorry I haven't been much help with debugging over the past few days ... been swamped and heads down.

I am no longer getting the error, but unfortunately I am not seeing any linting errors either.  Initially, I just updated the package, but then I uninstalled and reinstalled to attempt your fix and get the package working correctly.

For a brief time before the update, I did see a lint error on hover, but it seemed to be stuck and consistently reported the same (also first) error even after I fixed it.  A restart removed the stuck error, but then no linting errors would report.

It may help to assert that I am using the package as you intended:
* My templates are loading with `HTML (Handlebars)` grammar
* I am not sure if I should be expecting the templates to lint automatically or not.  I have tried toggling the switch via command and context menu, but neither seem to produce linting errors.

If I'm not missing an obvious dependency, this must be environmental with my machine, atom environment, styling, etc.

Sorry to pollute this issue and happy to open another issue if it would help.

Thanks again.
